Editorial: rename `holdings` to `heldValue` (#172)
tc39/proposal-weakrefs: README.md

This example shows usage of the whole `FinalizationGroup` API:
- An object can have a finalizer referenced by calling the `register` method of `FinalizationGroup`. In this case, three arguments are passed to the `register` method:
  - The object whose lifetime we're concerned with. Here, that's `this`, the `FileStream` object.

>   - A “heldValue” value, which is used to represent that object when cleaning it up in the finalizer. Here, the helf value is the underlying `File` object. (Note: heldValue should not have a reference to the weak target, as that would prevent the target from being collected.)

Reviewer: ```suggestion
  - A held value, which is used to represent that object when cleaning it up in the finalizer. Here, the held value is the underlying `File` object. (Note: the held value should not have a reference to the weak target, as that would prevent the target from being collected.)
```

  - An unregistration token, which is passed to the `unregister` method when the finalizer is no longer needed. Here we use `this`, the `FileStream` object itself, since `FinalizationGroup` doesn't hold a strong reference to the unregister token.
- The `FinalizationGroup` constructor is called with a callback as an argument. This callback is called with an iterator of the held values.

> The finalizer callback is called *after* the object is garbage collected, a pattern which is sometimes called "post-mortem". For this reason, a separate "heldValue" value is put in the iterator, rather than the original object--the object's already gone, so it can't be used.

Reviewer: ```suggestion
The finalizer callback is called *after* the object is garbage collected, a pattern which is sometimes called "post-mortem". For this reason, a separate held value is put in the iterator, rather than the original object--the object's already gone, so it can't be used.
```

This code uses a few features of the `FinalizationGroup` API:
- An object can have a finalizer referenced by calling the `register` method of `FinalizationGroup`. In this case, two arguments are passed to the `register` method:
  - The object whose lifetime we're concerned with. Here, that's the `Uint8Array`

>   - A “heldValue” value, which is used to represent that object when cleaning it up in the finalizer. In this case, the heldValue is an integer corresponding to the offset within the `WebAssembly.Memory` object.

Reviewer: ```suggestion
  - A held value, which is used to represent that object when cleaning it up in the finalizer. In this case, the held value is an integer corresponding to the offset within the `WebAssembly.Memory` object.
```
